### PR TITLE
Ocd 2544: update contact email throughout api

### PR DESCRIPTION
**Pull request by kekey1**

### CODE REVIEW

File: chpl/chpl-api/src/main/java/gov/healthit/chpl/SwaggerConfig.java [hunk 1]
```diff
@@ -37,31 +38,31 @@
...
     private ApiInfo apiInfo() {
-        LOGGER.info("get ApiInfo");
+        Contact contact = new Contact("CHPL Development Team. "
+                + "Please submit any questions using the Health IT Feedback Form and "
+                + "select the \"Certified Health IT Products List (CHPL)\" category.",
+                "https://www.healthit.gov/form/healthit-feedback-form",
+                "");
         return new ApiInfo("CHPL", "Certified Health IT Product Listing", "18.0.0", "http://terms/of/service.url",
-                "CHPL@ainq.com", "License Text", "https://github.com/chpladmin/chpl-api/blob/staging/LICENSE");
+                contact, "License Text", "https://github.com/chpladmin/chpl-api/blob/staging/LICENSE");
```
Comment (andlar) [inline chpl/chpl-api/src/main/java/gov/healthit/chpl/SwaggerConfig.java]: Should we/can we remove the terms of service link while we're in here? Or link to something that actually exists?

File: chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/UserManagementController.java [hunk 2]
```diff
@@ -337,7 +338,9 @@ public UserInvitation inviteUser(@RequestBody final UserInvitation invitation)
                 + "which will allow you to manage certified product listings on the CHPL. "
                 + "Please click the link below to create or update your account: <br/>"
                 + env.getProperty("chplUrlBegin") + "/#/registration/create-user/" + createdInvite.getInviteToken()
-                + "</p>" + "<p>If you have any questions, please contact the ONC CHPL Team at onc_chpl@hhs.gov.</p>"
+                + "</p>"
+                + "<p>If you have any issues completing the registration, "
+                + "please contact the ONC CHPL Team at onc_chpl@hhs.gov.</p>"
```
Comment (andlar) [inline chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/UserManagementController.java]: These _are_ HTML emails; is it worth it to make these mailto: links?

File: chpl/chpl-api/src/main/java/gov/healthit/chpl/SwaggerConfig.java (hunk 1 above)
Comment (kekey1) [reply chpl/chpl-api/src/main/java/gov/healthit/chpl/SwaggerConfig.java]: We can! Do we have terms of service online or should I just remove the link?

File: chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/UserManagementController.java (hunk 2 above)
Comment (kekey1) [reply chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/UserManagementController.java]: We can if you want. I've only ever sent these emails to my Ai account and looked at them in Outlook but in there they appear as mailto links anyway. I think it recognizes email addresses and displays them as such.

File: chpl/chpl-api/src/main/java/gov/healthit/chpl/SwaggerConfig.java (hunk 1 above)
Comment (andlar) [reply chpl/chpl-api/src/main/java/gov/healthit/chpl/SwaggerConfig.java]: I don't think we have any TOS anywhere. Unless ONC has some I'd suggest we just remove it

File: chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/UserManagementController.java (hunk 2 above)
Comment (andlar) [reply chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/UserManagementController.java]: Hmm. I hadn't noticed that. I forwarded one to my gmail account and it does that as a link too. I think it'd be a good idea to be explicit about it, but could go either way

**Pull request merged**